flambe/utils/config.py

```diff
@@ -0,0 +1,46 @@
+import os
+import re
+
+import jinja2
+
+
+def generate_config_from_template(template_path: str,
+                                  config_path: str,
+                                  remove_comments: bool = False,
+                                  **template_kwargs):
+    """
```

Comment:
This is missing a proper docstring. Please follow the numpy style:

```
Parameters
-----------
template_path: str
    description
config_path: str
 ...
```

Please also add typing to template_kwargs (Dics[str, str]?)